flixel/FlxObject.hx

```diff
@@ -1173,4 +1183,16 @@ class FlxObject extends FlxBasic
 	{
 		return allowCollisions = Value;
 	}
+	
+	private function set_path(path:FlxPath):FlxPath
+	{
+		if (this.path == path)
+			return path;
+		
+		if (this.path != null)
+			path.object = null;
```

Comment:
To me, `destroy()` implies setting `path` to null and letting it be garbage-collected. Like I mentioned earlier, the user might want to `restart()` the path.